src/test/java/io/opentracing/contrib/jfrtracer/JFRTracerTest.java

```diff
@@ -53,9 +53,22 @@ public void basicEvent() throws IOException {
 			// Generate span
 			tracer.buildSpan("test span").start().finish();
 
+            //to be removed
+            try {
+                Thread.sleep(100);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+
 			// Stop recording
 			List<FLREvent> events = JFRTestUtils.stopJfr(output);
 
+            try {
```

Comment:
Instead of catching interrupted exception everywhere, just throw it or add a helper method.